app/build.gradle.kts

```diff
@@ -77,4 +77,6 @@ dependencies {
     implementation("com.google.dagger:hilt-android:2.49")
     kapt("com.google.dagger:hilt-android-compiler:2.47")
     implementation("androidx.hilt:hilt-navigation-compose:1.2.0")
+
+    implementation ("androidx.navigation:navigation-compose:2.7.1")
```

Comment:
I recommend to use KSP (Kotlin Symbol Processing), which is a faster alternative to kapt. Both Dagger and Room support that.

I explain it here: https://www.youtube.com/watch?v=WVhgUGyUnhQ

Reply:
I'd recommend to use anything but Gson for JSON parsing. Gson is a Java library and has its issues with things like nullability and default values in Kotlin (for example, it can assign null values to nullable fields). Especially for Retrofit it's very easy to swap it out with a Kotlin library like Moshi or Kotlin serialization üòÑ